src/style/style.css

```diff
@@ -0,0 +1,306 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  font-family: cursive;
+}
+
+.nav {
+  background-color: #ffe200;
```

Comment:
- [ ] [OPTIONAL] great job guys 👍🏽 However, for a better UI(User Interface), I would suggest that you use a yellow colour with a higher gradient. Something like `#7e7004`. Currently, the whole landing page seems too bright.